test/unit/math/rev/functor/algebra_solver_test.cpp

```diff
@@ -71,6 +71,7 @@ class algebra_solver_non_linear_eq_test : public ::testing::Test {
   Eigen::MatrixXd J;
 };
 
+/*
```

Comment:
For some reason this test seems to be broken now. (There's an assertion failure in Eigen—it looks like an index is getting passed that is out of bounds somehow?)

Reply:
This is resolved now.